test/patterns/send_null/test_send_null.py

```diff
@@ -26,7 +26,7 @@ def test_simple_invocation(self):
 
     def test_more_method_invocations(self):
         lines = self.method_send_null_finder.value(Path(self.dir_path, 'SequenceFile.java'))
-        self.assertEqual(lines, [1186, 1201, 1217, 3285, 3298, 3367, 3537, 3550])
+        self.assertEqual(lines, [1097, 1186, 1201, 1217, 3285, 3298, 3367, 3537, 3550])
```

Comment:
why do we change asset expression?

Reply:
new cases were found since I was not looking for Literal only